api-surface-swift

```diff
@@ -75,6 +75,7 @@ public class Datadog
    public func set(tracesEndpoint: TracesEndpoint) -> Builder
    public func set(tracedHosts: Set<String>) -> Builder
    public func track(firstPartyHosts: Set<String>) -> Builder
+   public func trackURLSession(firstPartyHosts: Set<String> = []) -> Builder
```

Comment:
I'm not sure why this was added here.

Reply:
It's because we forgot to run `make api-surface` on the PR adding this API. Good to list it now 👍.